modules/osdk-ansible-k8s-module-inside-operator.adoc

```diff
@@ -0,0 +1,178 @@
+// Module included in the following assemblies:
+//
+// * operators/osdk-ansible.adoc
+
+[id='osdk-ansible-k8s-module-inside-operator-{context}']
+= Testing the k8s Ansible module inside an Operator
+
+After you are familiar using the `k8s` Ansible module locally, you can trigger
+the same Ansible logic inside of an Operator when a Custom Resource (CR)
+changes. This example maps an Ansible role to a specific Kubernetes resource
+that the Operator watches. This mapping is done in the Watches file.
+
+[id='osdk-ansible-k8s-module-inside-operator-testing-local-{context}']
+== Testing an Ansible-based Operator locally
+
+After getting comfortable testing Ansible workflows locally, you can test the
+logic inside of an Ansible-based Operator running locally.
+
+To do so, use the `operator-sdk up local` command from the top-level directory
+of your Operator project. This command reads from the `./watches.yaml` file and
+uses the `~/.kube/config` file to communicate with a Kubernetes cluster just as
+the `k8s` Ansible module does.
+
+////
+Possible .Prerequisites list item:
+
+This section assumes the developer has read the Ansible Operator user guide and has the proper dependencies installed.
+////
+
+.Procedure
+
+. Because the `up local` command reads from the `./watches.yaml` file, there are
+options available to the Operator author. If `role` is left alone (by default,
+`/opt/ansible/roles/<name>`) you must copy the role over to the
+`/opt/ansible/roles/` directory from the Operator directly.
++
+This is cumbersome because changes are not be reflected from the current
+directory. Instead, change the `role` field to point to the current directory
+and comment out the existing line:
++
+[source,yaml]
+----
+- version: v1alpha1
+  group: foo.example.com
+  kind: Foo
+  #  role: /opt/ansible/roles/Foo
+  role: /home/user/foo-operator/Foo
+----
+
+. Create a Custom Resource Definiton (CRD) and proper role-based access control
+(RBAC) definitions for the Custom Resource (CR) `Foo`. The `operator-sdk`
+command autogenerates these files inside of the `deploy/` directory:
++
+----
+$ kubectl create -f deploy/crds/foo_v1alpha1_foo_crd.yaml
```

Comment:
Since the OSDK is applicable to all Kube distros and not just OpenShift, I've so far left OSDK topics in OpenShift docs as `kubectl` while mentioning in the Prerequisites "can alternatively use `oc`" (though I have been uneven with including that caveat across all procedures...).

@hasbro17 I feel like we touched on this early on, but WDYT? @shawn-hurley?